tests/data/multimodel/container/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM 142577830533.dkr.ecr.us-west-2.amazonaws.com/ubuntu:16.04
```

Comment:
Is the dev account the decided account to host this image?

Reply:
Yes, this image is used only for our integration test, and it's only going to be used temporarily until [a permanent solution is available](https://aws.amazon.com/blogs/containers/advice-for-customers-dealing-with-docker-hub-rate-limits-and-a-coming-soon-announcement/):
> Within weeks, AWS will deliver a new public container registry that will allow developers to share and deploy container images publicly.

Reply:
The integ test failed at pulling this image from ECR. Perhaps it's because the tests are running in us-east-2?

Also will our regional canaries run this test as well? If so, will the regional accounts be able to access the image?